db/schema.rb

```diff
@@ -90,4 +90,20 @@
     t.datetime "password_reset_sent_at"
   end
 
+
+  create_view "register_search_results",  sql_definition: <<-SQL
```

Comment:
Is this a materialised view or not? In other words, is there any delay on reflecting changes when the underlying data changes?

Reply:
I answered this question reading scenic readme 👍